testing/unit_tests/test_testing.py

```diff
@@ -207,9 +207,10 @@ def test_inference_2d_microscopy(download_data_testing_test_files, transforms_di
                                                    ofolder=str(__output_dir__),
                                                    cuda_available=cuda_available)
 
-    assert len([x for x in __output_dir__.iterdir() if x.name.endswith(".nii.gz")]) == len(test_lst)
+    assert len([x for x in __output_dir__.iterdir() if x.name.endswith(".nii.gz")]) == len(test_lst) + 1
```

Comment:
note there seem to be an extra file stored in the folders when I use tmp folder instead of local tmp folder, don't know why